src/components/AccordionSection/readme.md

```diff
@@ -1,6 +1,11 @@
 ##### accordion base
 
 ```js
+import React from 'react';
+import Card from 'react-rainbow-components/components/Card';
+import Accordion from 'react-rainbow-components/components/Accordion';
+import AccordionSection from 'react-rainbow-components/components/AccordionSection';
```

Comment:
this is needed? I think that what is needed to import was components like GlobalHeader

Reply:
So sorry, I was forgetting to communicate with you for the new feature and breaking change of the react-styleguidist 9 version. For now no more global component, only the current component is accessible in the example context, or you can explicitly import everything, to make examples easier to copy into your app code.